### PR TITLE
Patch 2: Edited DNS_EnterMgt_Evtstrm_funcs ds and r

### PR DESCRIPTION
**Pull request by geethasathyamurthy**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: website/docs/r/function_trigger.html.markdown [hunk 1]
```diff
@@ -1,119 +1,110 @@
...
 
+## Argument reference
+Review the argument reference that you can specify for your resource. 
 
-## Argument Reference
-
-The following arguments are supported:
+- `feed` (List, Forces new resource, Optional)  A nested block to describe the feed.
+  
+  Namespace scheme for `feed`:
```
Comment (hkantare) [inline website/docs/r/function_trigger.html.markdown:74]: Namespace should be nested

File: website/docs/r/function_action.html.markdown [hunk 2]
```diff
@@ -1,154 +1,162 @@
...
-* `action_id` - Action ID	
-* `target_endpoint_url` - Target endpoint URL of the action.
 
-## Import
+## Argument reference
+Review the argument reference that you can specify for your resource. 
+ 
+- `exec` - (Required, List) A nested block to describe executable binaries.
+
+  Namespace scheme for `exec`:
```
Comment (hkantare) [inline website/docs/r/function_action.html.markdown:109]: Namespace scheme shd be Nested schema...simailrly down also

Comment (geethasathyamurthy) [reply website/docs/r/function_action.html.markdown]: updated the correction.

File: website/docs/r/function_trigger.html.markdown (hunk 1 above)
Comment (geethasathyamurthy) [reply website/docs/r/function_trigger.html.markdown]: updated the correction.

**Pull request merged**